projects/authentication/docs/user-access-token.md

```diff
@@ -278,6 +280,51 @@ There are two ways to check if your cached token is still valid:
 1.  Store the `expires_in` property included in the token response and the time that you requested the token internally in your application. Using these two parameters, you can calculate the expiration time of the token and request a new one when it is expired. Note that if you follow this approach, you should account for clock skew between your server and the APIs' servers, so it's best to already request a new token a couple of minutes before the cached one will expire.
 2.  Keep using the same cached token until you get a `401` response from an API endpoint, at which point you can request a new token and perform the failed request again with the new token. Note that you will need to set a maximum number of retries if you follow this approach, to prevent an infinite loop if there happens to be an issue that prevents you from getting a valid token.
 
+## Refresh tokens
```

Comment:
Since that is a different / older flow with the UiTdatabank JWT provider, I was thinking of documenting that in a separate guide in the UiTdatabank docs (or authentication docs but then still a different guide). Both how to do it + how to migrate to the standardized flows.